meta.yaml

```diff
@@ -0,0 +1,8 @@
+# conda metadata build recipe
```

Comment:
sorry what is this file?

Reply:
It's a [conda build recipe](https://docs.conda.io/projects/conda-build/en/stable/concepts/recipe.html) that we'll use to create a conda package